common/src/commonMain/kotlin/com/cuhacking/atlas/common/FeatureApi.kt

```diff
@@ -5,15 +5,18 @@ import com.cuhacking.atlas.db.Database
 import io.github.dellisd.spatialk.geojson.Feature
 import io.github.dellisd.spatialk.geojson.FeatureCollection
 import io.ktor.client.HttpClient
-import io.ktor.client.request.get
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
 import kotlinx.coroutines.withContext
+import kotlinx.datetime.Instant
+import kotlinx.datetime.toInstant
 
 class FeatureApi(
     private val database: Database,
     private val client: HttpClient,
     private val dispatchers: CoroutineDispatchers = CoroutineDispatchers
 ) {
-    private val dataCache = DataCache(dispatchers)
+    val dataCache = DataCache(dispatchers)
```

Comment:
What I think we can change this to is a constructor property. In our application, the data cache will be represented as a singleton object (there is only one instance per application) so instead of creating a new instance here, we can "provide" the instance to the `FeatureApi` via the constructor.

This also means that in the test code, you can create an instance of the data cache, then use it to test the API as well as use the instance to check if the test succeeded without having to expose any of `FeatureApi`'s private properties!